library/src/main/java/io/constructor/core/ConstructorIo.kt

```diff
@@ -484,14 +488,78 @@ object ConstructorIo {
      *          }
      *      }
      * ```
-     * @param request the search request object
+     * @param request the browse request object
      */
     fun getBrowseResults(request: BrowseRequest): Observable<ConstructorData<BrowseResponse>> {
         val encodedParams: ArrayList<Pair<String, String>> = getEncodedParams(facets = request.filters?.toList(), page = request.page, perPage = request.perPage, sortBy = request.sortBy, sortOrder = request.sortOrder, sectionName = request.section, hiddenFields = request.hiddenFields, hiddenFacets = request.hiddenFacets, groupsSortBy = request.groupsSortBy, groupsSortOrder = request.groupsSortOrder, variationsMap = request.variationsMap)
 
         return dataManager.getBrowseResults(request.filterName, request.filterValue, encodedParams = encodedParams.toTypedArray())
     }
 
+    /**
+     * Returns a list of browse results from a list of item IDs including filters, categories, sort options, etc.
+     * ##Example
+     * ```
+     *  runBlocking {
+     *      launch {
+     *          try {
+     *              val browseItemResults = constructorIo.getBrowseItemsResultsCRT("group_id", "888")
+     *              // Do something with browseItemsResults
+     *          } catch (e: Exception) {
+     *              println(e)
+     *          }
+     *      }
+     *  }
+     * ```
+     * @param facets  additional facets used to refine results
+     * @param itemIds the list of item ids to retrieve recommendations for (strategy specific)
```

Comment:
Need to update this to be about browse.

Also, not sure if the order matters, but should we put `itemIds` above `facets` to follow the param order in the function?